TestNEP17Transfer fail

[Run tests (macos-12, 1.22)](https://github.com/nspcc-dev/neo-go/actions/runs/8193301504/job/22406740884#logs)
```
2024-03-07T18:53:56.0599950Z     logger.go:146: 2024-03-07T18:53:49.692Z	INFO	Error encountered with rpc request	{"code": -510, "cause": "transaction has expired: ValidUntilBlock = 8, current height = 8", "method": "sendrawtransaction", "params": "[AGAlRl8q+ao7AAAAAIwwIAAAAAAACAAAAAHYzFqQTVJvyabKtVugLJpv54nJVgEA/cQDDSIDeyJuYW1lIjoiVGVzdCB2ZXJpZnkiLCJhYmkiOnsibWV0aG9kcyI6W3sibmFtZSI6Im9uTkVQMTFQYXltZW50Iiwib2Zmc2V0Ijo2LCJwYXJhbWV0ZXJzIjpbeyJuYW1lIjoiZnJvbSIsInR5cGUiOiJIYXNoMTYwIn0seyJuYW1lIjoiYW1vdW50IiwidHlwZSI6IkludGVnZXIifSx7Im5hbWUiOiJ0b2tlbiIsInR5cGUiOiJCeXRlQXJyYXkifSx7Im5hbWUiOiJkYXRhIiwidHlwZSI6IkFueSJ9XSwicmV0dXJudHlwZSI6IlZvaWQiLCJzYWZlIjpmYWxzZX0seyJuYW1lIjoib25ORVAxN1BheW1lbnQiLCJvZmZzZXQiOjIsInBhcmFtZXRlcnMiOlt7Im5hbWUiOiJmcm9tIiwidHlwZSI6Ikhhc2gxNjAifSx7Im5hbWUiOiJhbW91bnQiLCJ0eXBlIjoiSW50ZWdlciJ9LHsibmFtZSI6ImRhdGEiLCJ0eXBlIjoiQW55In1dLCJyZXR1cm50eXBlIjoiVm9pZCIsInNhZmUiOmZhbHNlfSx7Im5hbWUiOiJ2ZXJpZnkiLCJvZmZzZXQiOjAsInBhcmFtZXRlcnMiOltdLCJyZXR1cm50eXBlIjoiQm9vbGVhbiIsInNhZmUiOmZhbHNlfV0sImV2ZW50cyI6W3sibmFtZSI6Ik9uTkVQMTFQYXltZW50IiwicGFyYW1ldGVycyI6W3sibmFtZSI6ImZyb20iLCJ0eXBlIjoiSGFzaDE2MCJ9LHsibmFtZSI6ImFtb3VudCIsInR5cGUiOiJJbnRlZ2VyIn0seyJuYW1lIjoidG9rZW5JZCIsInR5cGUiOiJCeXRlQXJyYXkifSx7Im5hbWUiOiJkYXRhIiwidHlwZSI6IkFueSJ9XX1dfSwiZmVhdHVyZXMiOnt9LCJncm91cHMiOltdLCJwZXJtaXNzaW9ucyI6W10sInN1cHBvcnRlZHN0YW5kYXJkcyI6W10sInRydXN0cyI6W10sImV4dHJhIjpudWxsfQx3TkVGM25lby1nby0wLjkwLjAtdGVzdAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACkIQFcAA0BXAQR4eXp7VBTAcAwOT25ORVAxMVBheW1lbnRoUEGVAW9hQJX957wSwB8MBmRlcGxveQwU/aP6Q0bqUyolj8SX3a3bZDfJ/f9BYn1bUgFCDECPEtaj86sVe3IQvyt28yY9ieH40MWAfzx4EZpWdRwlv4/ASDHDQCrAYcdvk06KqLh+ZfYyk9v0r4UJ0Paah6BDKhEMIQKzYiv0AXvf4xfFiu1fTHU/IGt9uJYEb6fXdLvEv3+NwhFBntDcOg==]"}
2024-03-07T18:53:56.0643530Z Expired transaction (-510) - transaction has expired: ValidUntilBlock = 8, current height = 8
2024-03-07T18:53:56.0644520Z     executor.go:286: 
2024-03-07T18:53:56.0645510Z         	Error Trace:	/Users/runner/work/neo-go/neo-go/internal/testcli/executor.go:286
2024-03-07T18:53:56.0647140Z         	            				/Users/runner/work/neo-go/neo-go/internal/testcli/executor.go:367
2024-03-07T18:53:56.0648780Z         	            				/Users/runner/work/neo-go/neo-go/cli/nep_test/nep11_test.go:676
2024-03-07T18:53:56.0650640Z         	            				/Users/runner/work/neo-go/neo-go/cli/nep_test/nep17_test.go:182
2024-03-07T18:53:56.0651510Z         	Error:      	Received unexpected error:
2024-03-07T18:53:56.0652980Z         	            	Expired transaction (-510) - transaction has expired: ValidUntilBlock = 8, current height = 8
2024-03-07T18:53:56.0654020Z         	Test:       	TestNEP17Transfer
2024-03-07T18:53:56.0655180Z     logger.go:146: 2024-03-07T18:53:49.698Z	INFO	shutting down RPC server	{"endpoint": "127.0.0.1:49394"}
2024-03-07T18:53:56.0657180Z     logger.go:146: 2024-03-07T18:53:49.705Z	INFO	persisted to disk	{"blocks": 8, "keys": 250, "headerHeight": 8, "blockHeight": 8, "took": "2.197065ms"}
2024-03-07T18:53:56.0658530Z --- FAIL: TestNEP17Transfer (0.86s)
```

--- a/cli/txctx/tx.go
+++ b/cli/txctx/tx.go
@@ -75,7 +75,7 @@ func SignAndSend(ctx *cli.Context, act *actor.Actor, acc *wallet.Account, tx *tr
 			}
 			waitTime := time.Since(promptTime)
 			// Compensate for confirmation waiting.
-			tx.ValidUntilBlock += uint32((waitTime.Milliseconds() / int64(ver.Protocol.MillisecondsPerBlock))) + 1
+			tx.ValidUntilBlock += uint32(waitTime.Milliseconds()/int64(ver.Protocol.MillisecondsPerBlock)) + 2
 		}
 		var (
 			resTx util.Uint256

